app/Http/Controllers/UserController.php

```diff
@@ -44,9 +44,22 @@ public function displayProfile(User $userId)
 
     public function followUser(User $profileId)
     {
-        $currentUser = Auth::user();
 
+        $currentUser = Auth::user();
         $profileId->followers()->attach($currentUser->id);
+
+        $fId= DB::table('followers')->orderBy('created_at','desc')->first()->id;
```

Comment:
improve the name of the variable. Variable names should have more clear intentions